src/main/java/org/recordrobotics/charger/commands/manual/ManualClaw.java

```diff
@@ -0,0 +1,51 @@
+package org.recordrobotics.charger.commands.manual;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import org.recordrobotics.charger.subsystems.Claw;
+import org.recordrobotics.charger.control.IControlInput;
+
+
+public class ManualClaw extends CommandBase {
+
+	private Claw _claw;
+	private IControlInput _controls;
+
+	private static final double TURN_SPEED = 0.3;
+
+	public ManualClaw(Claw claw, IControlInput controls) {
+		if (claw == null) {
+			throw new IllegalArgumentException("claw is null");
+		}
+		if (controls == null) {
+			throw new IllegalArgumentException("Controls is null");
+		}
+
+		_claw = claw;
+		_controls = controls;
+		addRequirements(claw);
+	}
+
+	@Override
+	public void execute() {
+		if(_controls.getClawTurn() > 0){
+			if(_claw.getPosition() <= Claw.CLAW_CUBE){
+				_claw.turn(TURN_SPEED);
+			}else{
+				_claw.turn(0);
+			}
+		}else if(_controls.getClawTurn() < 0){
+			if(_claw.getPosition() <= Claw.CLAW_CONE){
+				_claw.turn(TURN_SPEED);
+			}else{
+				_claw.turn(0);
+			}
+		}else{
+			_claw.turn(-TURN_SPEED);
+		}
```

Comment:
Suppose this scenario: I press the cone button, then press the cube button and then release cone button.
I don't think this code would handle this.

Reply:
done